Don't import deprecated django.utils.datetime_safe
--- python-django-tastypie.orig/tastypie/compat.py
+++ python-django-tastypie/tastypie/compat.py
@@ -18,7 +18,7 @@ else:
 
 # Django 5.0 eliminated the former datetime_safe function, this provides
 # some level of backwards compatability for existing tastypie use cases
-if django.VERSION < (5, 0):
+if django.VERSION < (4, 0):
     from django.utils import datetime_safe  # noqa: F401
 else:
     import datetime as datetime_safe  # noqa: F401
